File: docs/extensibility/codesnippet/CSharp/walkthrough--displaying-signature-help_14.cs
    public string Content
    {
        get { return (m_content); }
        internal set { m_content = value; }
    }

    public string Documentation
    {
        get { return (m_documentation); }
        internal set { m_documentation = value; }
    }

    public ReadOnlyCollection<IParameter> Parameters
    {
        get { return (m_parameters); }
        internal set { m_parameters = value; }
    }

    public string PrettyPrintedContent
    {
        get { return (m_printContent); }
        internal set { m_printContent = value; }
    }

File: docs/extensibility/codesnippet/VisualBasic/walkthrough-displaying-signature-help_14.vb
    ReadOnly Property Content() As String Implements ISignature.Content
        Get
            Return (m_content)
        End Get
    End Property

    ReadOnly Property Documentation() As String Implements ISignature.Documentation
        Get
            Return (m_documentation)
        End Get
    End Property

    ReadOnly Property Parameters() As ReadOnlyCollection(Of IParameter) Implements ISignature.Parameters
        Get
            Return (m_parameters)
        End Get
    End Property

    ReadOnly Property PrettyPrintedContent() As String Implements ISignature.PrettyPrintedContent
        Get
            Return (m_printContent)
        End Get
    End Property